### PR TITLE
Add Konachan.app v1.0

### PR DESCRIPTION
**Pull request by yaqinking**

### CODE REVIEW

File: Casks/konachan.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'konachan' do
+  version '1.0'
+  sha256 '248ff1db7bf9ca3d8148253c7b7053c89982419dd1481e103780b8dfb9aa1e11'
+
+  url 'https://github.com/yaqinking/Konachan/releases/download/v1.0/Konachan.app.zip'
```
Comment (fanquake) [inline Casks/konachan.rb]: You can use #{version} in the URL i.e download/v#{version}/Konachan

Comment (yaqinking) [inline Casks/konachan.rb]: When I replace `1.0` to `#{version}` It display an error
```
Error: Cask 'konachan' definition is invalid: 'url' stanza failed with: bad URI(is not URI?): https://github.com/yaqinking/Konachan/releases/download/v#{version}/Konachan.app.zip
```
What's wrong with me?

Comment (yaqinking) [inline Casks/konachan.rb]: I replace ' to "
It works.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in the below referenced pull request. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**